deployments/demos/zenoh-bridge/README.md

```diff
@@ -1,3 +1,37 @@
-# zenoh-bridge demo
+# Zenoh Bridge Demo
```

Comment:
Instead of updating the README, it would be better to update the documentation directly, while people will refer to that first.
https://autowarefoundation.github.io/openadkit/deployments/zenoh-bridge/